packages/core/package.json

```diff
@@ -34,7 +34,7 @@
     "@ucanto/transport": "^9.1.1",
     "@web3-storage/access": "^18.4.0",
```

Comment:
could we also update access to latest? Not sure if there are any implications there